src/main/java/Parser.java

```diff
@@ -0,0 +1,156 @@
+public class Parser {
+    private static void checkValidDeadline (String command) throws WrongInputFormat, MissingEntries, MissingTaskName {
+        String[] commandWords = command.split("/");
+
+        if (commandWords.length != 2) {
+            throw new WrongInputFormat();
+        } else {
+            if (commandWords[0].length() < 10) {
```

Comment:
Try not to use magic literals e.g. magic numbers and magic string, consider creating a constant and use the constant in your code instead.